R/get_data.R

```diff
@@ -12,8 +12,11 @@
 #'  * `"header"`: plot-level information.
 #'  * `"DT"`: a list of species and relative cover in each vegetation plot.
 #'  * `"CWM_CWV"`: community-weighted means and variances for 18 traits.
-#' @param metadata If `TRUE` (the default), metadata will be downloaded.
 #'
+#' @param metadata If `TRUE` (the default), metadata will be downloaded.
+#' @param additional_data If `FALSE` (the default) additional data downloaded
+#'   from repository will be discarded.
```

Comment:
Hi Andrew, I consider it is useful to have them as two different arguments because I think downloading the metadata should be set as default (metada = TRUE) and the additional files not (auxiliary_files = FALSE). I believe it is important because the metadata is an essential part of the dataset, while the extra files are not. I had changed my pull request to reflect this.